### PR TITLE
Fix jumping carousel image height

### PR DESCRIPTION
**Pull request by Runeii**
When carousel images' aspect ratios do not match, the carousel height will vary. This can result in the images overlapping the number/pagination indicator, or a varying margin height above the pagination.

As agreed, the first slide should dictate the slide of the carousel. This MR ensures successive slides only fill the same aspect ratio space as the first slide.

**Steps to test**
1.  Visit the Jaquemus project page
2. Image carousel slides should all match each other

### CODE REVIEW

File: src/components/Carousel/Carousel.module.css [hunk 1]
```diff
@@ -4,20 +4,22 @@
...
 }
 
 .image {
   width: 100%;
   height: 100%;
+  overflow: hidden;
   opacity: 0;
 
-  &:not(:first-of-type) {
-    position: absolute;
```
Comment (jondujaka) [inline src/components/Carousel/Carousel.module.css:19]: have to say, this feels a bit dirty. A position absolute image inside a position absolute div. Do we need the wrapper `div` at all?

Comment (Runeii) [reply src/components/Carousel/Carousel.module.css]: Yeah, good point, it was basically building on top of a pretty messy block of code with even more mess. Have taken the time to work out what is going on and simplify.

File: src/components/Carousel/Carousel.module.css [hunk 2]
```diff
@@ -32,6 +30,7 @@
     transition: opacity 1.2s;
```
[the remark this answers is not in the document]
Comment (Runeii) [reply src/components/Carousel/Carousel.module.css:30]: Nah, it ended up being a pointer-event fix that achieved the same thing. I left it in, it's below here and looks like this:

```
  &[data-visible="false"] {
    pointer-events: none;
  }
```

[the remark this answers is not in the document]
Comment (jondujaka) [reply src/components/Carousel/Carousel.module.css:30]: Ah cool cool, yea that seems good then

**Pull request merged**